src/fidesops/core/config.py

```diff
@@ -39,8 +39,8 @@ class ExecutionSettings(FidesSettings):
     TASK_RETRY_BACKOFF: int
     REQUIRE_MANUAL_REQUEST_APPROVAL: bool = False
     MASKING_STRICT: bool = True
-    CELERY_BROKER_URL: str
-    CELERY_RESULT_BACKEND: str
+    CELERY_BROKER_URL: str = "redis://:testpassword@redis:6379/1"
+    CELERY_RESULT_BACKEND: str = "redis://:testpassword@redis:6379/1"
```

Comment:
Should we move this back to the fidesops.toml file?

Reply:
I thought about this. It can always be overridden from `fidesops.toml`, but the reality is that since the webserver always depends on the Redis cache it's safe to include as a default.